docs/application.html

```diff
@@ -271,7 +272,7 @@ <h1>XtkSchemaNode</h1>
     <tr><td><b>label</b></td><td>The label (i.e. human readable, localised) name of the node.</td></tr>
     <tr><td><b>labelLocalizationId</b></td><td>The translation id of the label of the node.</td></tr>
     <tr><td><b>name</b></td><td>The name of the node (internal name)</td></tr>
-    <tr><td><b>nodePath** </td><td>The xpath of the node
+    <tr><td><b>nodePath**</b></td><td>The xpath of the node
```

Comment:
same as before

Reply:
The <b> tag was not closed before (</b> was missing).